packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl

```diff
@@ -1,126 +1,109 @@
-# Customize inbound failsafe rules
-# https://docs.projectcalico.org/v3.5/getting-started/bare-metal/policy/failsafe
+# General
+# =======
+---
+# Disable default failsafe inbound rules
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  failsafeInboundHostPorts: []
+
+# Host endpoints
+# ==============
+---
+${controller_host_endpoints}
+---
+${worker_host_endpoints}
+
+# Firewall policy
+# ===============
+---
+# SSH access
 apiVersion: crd.projectcalico.org/v1
 kind: GlobalNetworkPolicy
 metadata:
-  name: failsafe-ingress
+  name: ssh
 spec:
+  selector: has(host-endpoint)
   preDNAT: true
   applyOnForward: true
-  selector: has(host-endpoint)
   order: 0
   ingress:
-  # SSH access
   - action: Allow
     protocol: TCP
     source:
       nets: ${management_cidrs}
     destination:
       ports: [22]
-  # DHCP access
-  - action: Allow
-    protocol: UDP
-    source:
-      nets: ${node_public_ips}
```

Comment:
This variable is not used anymore, so it can be removed from the vars this template has and the terraform variable for this.

It will break existing users, as they will need to remove that bar, we might want to decide wether this is fine in a platform in beta or if terraform provides an easy way to put a warning for the deprecated variable and remove the variable later.

Reply:
Good point, thanks.

I don't see why this would break anything. `node_public_ips` is a local variable just for this template. We are not touching the module's vars.

Reply:
Sorry I wasn't clear. With this patch that variable is not needed in the template, and in the template is the only place this variable is used. Therefore, we *can* remove the variable from the terraform module too.

If we do remove the variable from the tf module, it willbreak existing users and:

> It will break existing users, as they will need to remove that bar, we might want to decide wether this is fine in a platform in beta or if terraform provides an easy way to put a warning for the deprecated variable and remove the variable later.